### DISCUSSION TITLE
Date AND Time?

### DISCUSSION THREAD

**Discussion by nofue** [sqlpage/SQLPage#607] [Q&A]
The db field is defined as TIMESTAMP. But how do I get Date AND Time in a form? I can make fields either date or time -- do I need two fields and CONCAT these? Even in the "flight booking" example it's only a date -- having no time for departure won't work well, right?

**Comment by lovasoa** [comment]
You can use an input of type [datetime-local](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local). 
In a database that has a native TIMESTAMP type, you can simply use it as value.

In SQLite, if your timestamp is stored as a number, you can format it as a datetime using [strftime](https://www.sqlite.org/lang_datefunc.html).

**Reply by nofue** [reply]
Thanks for the hints -- for the time being I went with two fields, DATE and TIME and where I need both values together, such as when exporting CSV, I simply CONCAT the fields.
